config.c

```diff
@@ -377,65 +446,74 @@ bool apply_superset_style(
 	return true;
 }
 
-static bool apply_config_option(struct mako_config *config, const char *name,
-		const char *value) {
+static bool apply_surface_option(struct mako_style *surface,
```

Comment:
Looks like this can be merged back into `apply_style_option` now since they get moved over to the surface configs later.